bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml

```diff
@@ -88,6 +88,18 @@ spec:
     spec:
       clusterPermissions:
       - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - serviceaccounts
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
```

Comment:
BTW why these permissions? I didn't notice the service account manipulation.

Reply:
Removed them, it will be added in the next PR.